src/python/WMCore/ReqMgr/Service/Request.py

```diff
@@ -487,6 +499,32 @@ def _handleOnlyStateTransition(self, workload, request_args, dn):
         report = self.reqmgr_db_service.updateRequestStatus(workload.name(), req_status, dn)
         return report
 
+    def _updateNATS(self, workload, request_args):
```

Comment:
Valentin, not really a review, but just a comment on something that caught my attention.

Why did you chose this module to implement these updates? I think StdBase, same as done for old dashboard:
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/WMSpec/StdSpecs/StdBase.py#L248

would make much more sense.

Reply:
We need two monitoring, one for ReqMgr transitions and one for workflows. This was requested by McM group. The former is implemented in ReqMgr, the later in WMRuntime.

I asked originally about where it should be done, since I didn't heard from you I implemented where I new it will be possible. You better know the entire system and I would rely on your expertise to guide where the changes should be added.

The change I made in ReqMgr I tested with reqmgr2 service and it seems to work fine for transitions. If you think it should be relocated please provide full details.